commands/test/sampleCmd.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
The function shouldn't be `async` and why did you delete all the example attributes?

Reply:
welp my bad, I fixed that too